product/interfaces/templatefunctions/findProducts.go

```diff
@@ -33,25 +34,31 @@ func (tf *FindProducts) Func(ctx context.Context) interface{} {
 
 	/*
 		widgetName - used to namespace widget - in case we need pagination
-		config - map with certain keys - used to specifiy th searchRequest better
+		config - map with certain keys - used to specify the searchRequest better
 	*/
-	return func(namespace string, configs ...map[string]string) *application.SearchResult {
-		var searchConfig, keyValueFilters, filterConstrains map[string]string
+	return func(namespace string, configs ...pugjs.Map) *application.SearchResult {
+		var searchConfig, filterConstrains map[string]string
+		var keyValueFilters map[string][]string
 
 		if len(configs) > 0 {
-			searchConfig = configs[0]
+			searchConfig = configs[0].AsStringMap()
 		} else {
 			searchConfig = make(map[string]string)
 		}
 
```

Comment:
Suggestion: Initialise searchConfig as a map instead of just declaring it and remove the else case
I think this provides better readability as there is just a default case and an if